CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
+
+Swift 4.2
+---------
+
+* [SE-209](https://github.com/apple/swift-evolution/blob/master/proposals/0209-package-manager-swift-lang-version-update.md)
+
+The `swiftLanguageVersions` property no long takes its Swift language versions via
```

Comment:
Can you put a tab here and in rest of SE descriptions so it matches the style used in Swift 3.0 section?